### PR TITLE
try build registry on gh-actions - first attemptGhactions

### PR DESCRIPTION
**Pull request by maelle**
I wanted to see the diff @sckott :-)

### CODE REVIEW

File: inst/automation/git_urls.rb [hunk 1]
```diff
@@ -65,7 +65,7 @@ def gh_default_branch(owner_repo)
 
 # add other repos (those not in ropensci or ropenscilabs)
 nms = ["package", "url"]
-url = 'https://raw.githubusercontent.com/ropenscilabs/makeregistry/master/inst/automation/not_transferred.txt'
+url = 'https://raw.githubusercontent.com/ropensci-org/makeregistry/master/inst/automation/not_transferred.txt'
```
Comment (maelle) [inline inst/automation/git_urls.rb:68]: why is this read from GitHub and not from the source?

File: inst/automation/last_commits.R [hunk 2]
```diff
@@ -0,0 +1,31 @@
+library(gert)
```
Comment (maelle) [inline inst/automation/last_commits.R:1]: shouldn't this be a function in the package that'd be called from your script? Just so we're sure dependencies like gert are listed in DESCRIPTION?

File: inst/automation/pull_other.sh [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 rm -rf */
 
-curl https://raw.githubusercontent.com/ropenscilabs/makeregistry/master/inst/automation/not_transferred.txt > not_transferred.txt
+curl https://raw.githubusercontent.com/ropensci-org/makeregistry/master/inst/automation/not_transferred.txt > not_transferred.txt
```
Comment (maelle) [inline inst/automation/pull_other.sh:5]: This could be read from the source

File: inst/automation/git_urls.rb (hunk 1 above)
Comment (sckott) [reply inst/automation/git_urls.rb:68]: just laziness i think

File: inst/automation/last_commits.R (hunk 2 above)
Comment (sckott) [reply inst/automation/last_commits.R:1]: hmm, but if gert isn't needed in makeregistry, then doesn't seem like we should include it as a dependency

File: inst/automation/pull_other.sh (hunk 3 above)
Comment (sckott) [reply inst/automation/pull_other.sh:5]: will change that

File: inst/automation/last_commits.R (hunk 2 above)
Comment (maelle) [reply inst/automation/last_commits.R:1]: but how does one get an overview of all dependencies? and why couldn't this be another function? genuine questions, I am not that stubborn :-)

Comment (sckott) [reply inst/automation/last_commits.R:1]: it was just easier to have a one of script, we could bring it into the pkg

**Pull request merged**